modules/beamdyn/src/BeamDyn_Subs.f90

```diff
@@ -1045,22 +1046,59 @@ SUBROUTINE BD_ComputeIniNodalCrv(e1, phi, cc, ErrStat, ErrMsg)
    CHARACTER(ErrMsgLen)           :: ErrMsg2       ! Temporary Error message
    CHARACTER(*), PARAMETER        :: RoutineName = 'BD_ComputeIniNodalCrv'
 
+   INTEGER(IntKi)                 :: i,j
+   REAL(BDKi)                     :: rtwist(3,3),q0,q1,q2,q3,identity(3,3),qt(3,3)
+
    ! Initialize ErrStat
    ErrStat = ErrID_None
    ErrMsg  = ""
 
    PhiRad = phi*D2R_D  ! convert to radians
 
-      ! Note that e1 corresponds with the Z-axis direction in our formulation.  For Beam theory, this would be the x-axis.
-   Rr(:,3)  =  e1(:)
-
-   e2(3)    = -(e1(1)*COS(PhiRad) + e1(2)*SIN(PhiRad))/e1(3)
-   Delta    = SQRT(1.0_BDKi + e2(3)*e2(3))
-   e2(1)    = COS(PhiRad)
-   e2(2)    = SIN(PhiRad)
-   e2       = e2 / Delta
-   Rr(:,1)  = e2
-   Rr(:,2)  = Cross_Product(e1,e2)
+   ! Note that e3 corresponds with the Z-axis direction in our formulation.  For Beam theory, this would be the x-axis.
+   !Rr(:,3)  =  e3(:)
```

Comment:
@michaelasprague Do we need to keep this commented code?

Reply:
I think we can safely toss lines 1058:1068 -- there's no reason in my opinion to keep them.

Reply:
I removed them.